application/config/config-defaults.php

```diff
@@ -227,6 +227,25 @@
 // allow these users to be able to use Javascript etc. .
 $config['filterxsshtml'] = true;
 
+// disablescriptwithxss
+// Allow update of script in question
+// true : Default : follow XSS rules
+// false : allowed for all
+$config['disablescriptwithxss'] = true;
+
+// forcedfilterxss
+// Enables filtering of suspicious html tags in survey, group, questions
+// and answer texts in the administration interface
+// Even super-administrator have this restriction.
+// This settings can not be updated by GUI interface
+$config['forcedfilterxss'] = false;
+
+// superadminenablescript
+// Totally disable question script update
+// Remove the script part in question edition
+// Used only if forcedfilterxss and disablescriptwithxss are true
+$config['superadminenablescript'] = false;
```

Comment:
I think this should be a global option.
The script field may be disabled by a personal setting, it is missing a global setting, yet.

Reply:
This settings can disable superadmin to have script … by security : disallow ALL user to add scripts.

If you enable `forcedfilterxss` : superadmin can not add script inside question->text (or anywhere) => Must not be in GUI.
But if you enable at same time `disablescriptwithxss` : superadmin can not add (or update) script. We must leave allowing this ONLY in php file : Some server admin can want to disable script for superadmin too (see some bug report about XSS in mantis : XSS by superadmin).

Reply:
This line:

>  // Totally disable question script update

does not make sense with regard to the option. Maybe change to:

>  // If false, will totally disable question script update

and of course also include a description if set to true.